presto-main/src/main/java/com/facebook/presto/operator/scalar/ApplyFunction.java

```diff
@@ -93,11 +92,6 @@ public ScalarFunctionImplementation specialize(BoundVariables boundVariables, in
 
     public static Object apply(Object input, UnaryFunctionInterface function)
     {
-        try {
-            return function.apply(input);
-        }
-        catch (Throwable throwable) {
-            throw Throwables.propagate(throwable);
-        }
+        return function.apply(input);
```

Comment:
In case you wonder why code was written like this, `function` used to be a `MethodHandle`. cc @wenleix